addons/crm/tests/test_performances.py

```diff
@@ -92,13 +92,14 @@ def test_assign_perf_no_duplicates(self):
         leads.flush()
 
         with self.with_user('user_sales_manager'):
-            with self.assertQueryCount(user_sales_manager=589):  # 584-585 generally, sometimes 589
-                self.env['crm.team'].browse(self.sales_teams.ids)._action_assign_leads(work_days=2)
-
+            with self.profile(collectors=['sql']):
```

Comment:
Is this supposed to remain here?

Reply:
No I need to remove that before merging :) I left it with the idea to run multibuilds on runbot and hopefully have a profile with all queries if a count breaks.